chain/wasm/src/builders/witness_builder.rs

```diff
@@ -0,0 +1,227 @@
+use wasm_bindgen::prelude::{wasm_bindgen, JsError};
+use cml_core_wasm::impl_wasm_conversions;
+use cml_crypto_wasm::{
+    DatumHash, Ed25519KeyHash, ScriptHash
+};
+use crate::{
+    Script,
+    byron::ByronAddress,
+    crypto::{BootstrapWitness, Vkeywitness},
+    transaction::{TransactionWitnessSet},
+    plutus::{PlutusData, Redeemer},
+    NativeScriptList, PlutusV1ScriptList, PlutusV2ScriptList, RedeemerList, PlutusDataList, Ed25519KeyHashList, address::RewardAddress,
+};
+
+use super::redeemer_builder::{RedeemerWitnessKey};
+
+#[wasm_bindgen]
+#[derive(Debug, Clone)]
+pub struct PlutusScriptWitness(cml_chain::builders::witness_builder::PlutusScriptWitness);
+
+impl_wasm_conversions!(cml_chain::builders::witness_builder::PlutusScriptWitness, PlutusScriptWitness);
+
+#[wasm_bindgen]
+impl PlutusScriptWitness {
+    // pub fn script(&self) -> Option<PlutusScript> {
```

Comment:
why commented?

Reply:
It's how the rust part was. I think it slipped through before and we didn't end up needing to port it over in the utility code. I can either write the appropriate function in the rust builder (and add the wrapper here), or just remove the commented out function from both.